crates/red_knot_python_semantic/resources/mdtest/type_properties/is_assignable_to.md

```diff
@@ -331,6 +331,25 @@ static_assert(is_assignable_to(Intersection[LiteralString, Not[Literal[""]]], No
 static_assert(is_assignable_to(Intersection[LiteralString, Not[Literal["", "a"]]], Not[AlwaysFalsy]))
 ```
 
+## Generics
+
+### Inheriting from a specialized generic class
+
+```py
+from knot_extensions import is_assignable_to, static_assert
+from typing import Any
+
+class A[T]:
+    pass
+
+class B(A[int]):
+    pass
+
+static_assert(is_assignable_to(B, A[int]))
+static_assert(not is_assignable_to(B, A[bool]))
```

Comment:
This assertion is technically wrong and should get a TODO comment: see my other comment about variance.

Reply:
Done (in new tests)